oximeter/oximeter/schema/instance-network-interface.toml

```diff
@@ -0,0 +1,86 @@
+format_version = 1
+
+[target]
+name = "instance_network_interface"
+description = "A network interface attached to a virtual machine instance"
+authz_scope = "fleet"
+versions = [
+    { version = 1, fields = [ "interface_id", "instance_id", "project_id", "silo_id", "sled_id", "sled_model", "sled_revision", "sled_serial" ] },
```

Comment:
I talked with @ahl about the similar changes I've made to publish virtual disk statistics in https://github.com/oxidecomputer/omicron/pull/6420. He brought up legitimate points about including the "sled_*" fields here. The data is not particularly actionable, especially to customers in the developer role, but even operators. It's also a bit of an implementation detail -- for example, one would need to know that the hosting sled ID could change as an instance migrates, which would mean we'd need to handle that when showing data in the console, for example. It also complicates the very tentative authz model we have in mind, which involves being able to limit visibility of timeseries, but not of individual fields on a timeseries, to different groups of people. Last, we can technically get some of this in most cases from other database state, such as the migration or inventory tables.

All that is to say, I think we should probably leave _off_ the physical sled identifiers for timeseries that are about virtual resources, or things that many or all customers can see. E.g., I'm probably going to remove this from the vCPU usage data I added recently.

I'm open to being convinced otherwise, if you have strong reason to believing we should keep it. @rmustacc probably has other thoughts here too :)

Reply:
Makes sense to me honestly. I was kinda shoehorning it in b/c it was available previously.

Reply:
So remove all `sled_*` fields right?